change/@fluentui-react-breadcrumb-preview-2f782c5e-a8aa-467f-a0cd-fece7f7e03e0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
```suggestion
  "type": "major",
```

This is actually a breaking change

Reply:
Agree. Why there were no this option while `yarh change` command?